R code/Droughtnet_cleaning.qmd

```diff
@@ -0,0 +1,280 @@
+---
+title: "Droughtnet cleaning code"
+format: html
+editor: visual
+---
+
+```{r}
+library(tidyverse)
+library(readr)
+library(here)
+#install.packages("here")
+
+DURIN <- read_csv("C:/Users/User/Desktop/Durin_data/R code/DURIN_clean.csv", col_types = cols(calluna_shoot_type = col_factor(levels = c("Long", "Short"))),
```

Comment:
You should not use absolute path. If you work in an R project, put your data inside the R project in a folder called data or raw_data and then you only need: data/...